src/main/api/apigClient.js

```diff
@@ -53,7 +53,7 @@ apigClientFactory.newClient = function (config) {
 
```

Comment:
Your commit is based off a really old version... can you rebase it on a more recent version?
We moved the api to: 
`src/main/ionic/peers/src/assets/api`